pygsti/baseobjs/basis.py

```diff
@@ -547,8 +547,7 @@ def is_normalized(self):
         """
         if self.elndim == 2:
             for i, mx in enumerate(self.elements):
-                t = _np.trace(_np.dot(mx, mx))
-                t = _np.real(t)
+                t = _np.linalg.norm(mx.ravel(), 2) # == sqrt(tr(mx mx))
```

Comment:
I suppose the ravel isn't needed. I only added it because I prefer to be explicit when specifying the norm, and ``norm(arg.ravel(), 2)`` will always work for the Frobenius norm, while ``norm(arg, 'fro')`` will only work when ``arg.ndim == 2``. 

I'll change to ``norm(arg)`` here and elsewhere.